docs/src/test/translator.txt

```diff
@@ -0,0 +1,94 @@
+GPL 0002 placerBase.cpp:778        DBU: {} INFO https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/workspace/src/gpl/src/placerBase.cpp#L778
+GPL 0003 placerBase.cpp:806        SiteSize: {} {} INFO https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/workspace/src/gpl/src/placerBase.cpp#L806
```

Comment:
Will this test fail if line numbers change?

Reply:
Apologies. This `count_outfiles` test will be removed. I added a prototype 2 new local tests to src/ant, and modified the docs/src/test to be depending on static files instead. Could you have a look?
ant_man_tcl_check checks for man,proc, readme counts
ant_readme_msgs_check checks for readme counts and messages count.

Reply:
This is now static and independent from the original gpl `messages.txt`, so no.